_data/master/navbars/introduction.yml

```diff
@@ -5,3 +5,7 @@ order: 0
 toc:
 - title: Introduction
   path: /introduction/
+- title: Component Architecture
```

Comment:
Maybe these should be under a container section so that we can then have an OpenStack section too.